docs/OnlineTutorial/Lemmas.md

```diff
@@ -461,27 +455,27 @@ if we expanded `count(a)`. Specifically:
 
 ``` {.editonly}
 method m2(a: seq<bool>, b:seq<bool>)
-   requires |a| > 0
+  requires |a| > 0
 {
-   assert count(a) == count([a[0]]) + count(a[1..]);
+  assert count(a) == count([a[0]]) + count(a[1..]);
 }
 function count(a: seq<bool>): nat
 {
-   if |a| == 0 then 0 else
-   (if a[0] then 1 else 0) + count(a[1..])
+  if |a| == 0 then 0 else
+    (if a[0] then 1 else 0) + count(a[1..])
```

Comment:
```suggestion
  if |a| == 0 then 0 else
  (if a[0] then 1 else 0) + count(a[1..])
```

Reply:
Yes, as in my previous comment, since the "then" and the "else" are on the same line, adding indentation would not make it more legible if there are more nested if then else.